client/web/src/search/results/streaming/progress/StreamingProgressCount.tsx

```diff
@@ -4,6 +4,19 @@ import * as React from 'react'
 import { pluralize } from '../../../../../../shared/src/util/strings'
 import { StreamingProgressProps } from './StreamingProgress'
 
+const abbreviateNumber = (number: number): string => {
+    if (number < 1e3) {
+        return number.toString()
+    }
+    if (number >= 1e3 && number < 1e6) {
+        return (number / 1e3).toFixed(1) + 'k+'
+    }
+    if (number >= 1e6 && number < 1e9) {
+        return (number / 1e6).toFixed(1) + 'm+'
+    }
+    return (number / 1e9).toFixed(1) + 'b+'
+}
+
```

Comment:
I believe this would accomplish a similar outcome without having to do manual bounds-checking:

```suggestion
const abbreviateNumber = (number: number): string =>
    new Intl.NumberFormat('en-US', { notation: 'compact' }).format(number) + '+'
```

Reply:
Unfortunately `notation: 'compact'` does not work in Safari :(